quizcon/settings_shared.py

```diff
@@ -48,7 +47,11 @@
     'navigation': True,
     'new_tab': True,
     'frame_width': 1024,
-    'frame_height': 1024
+    'frame_height': 1024,
+    'assignments': {
```

Comment:
Note: This configuration can be removed once we update to the latest `django-lti-provider`.